charts/datarepo-api/templates/api-deployment.yaml

```diff
@@ -54,185 +57,11 @@ spec:
         - containerPort: {{ .Values.image.port }}
         - containerPort: 5005
         env:
-        {{- if .Values.env.serverContextpath }}
-        - name: SERVER_CONTEXTPATH
-          value: {{ .Values.env.serverContextpath | quote }}
+        {{- if .Values.env }}
```

Comment:
Can you add a comment that this dynamically populates environment variables.

Reply:
add comments